cni/network/network_windows.go

```diff
@@ -264,59 +263,52 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig, isIPv6Enabled bool) []p
 			protocol = policy.ProtocolUdp
 		}
 
+		hostIP := net.ParseIP(mapping.HostIp)
```

Comment:
Can you use `netip.ParseAddr`? It returns an error which makes it clearer that the input may have been malformed, and the return has clearer methods such as `Is4()` or `Is6()` that also make the switch below cleaner.